liveblog/liveblog
# [LBSD-2785] - Implement features service to check for subscription limits

Implements a new service, both backend and frontend, that will be used to check if certain features are enabled or to check the limitations of the current user's subscription. 

### What has changed
- Added a new python service class that fetches the settings from database and does a simple caching logic to avoid hitting database too often
- Implemented a new simple api endpoint to retrieve instance settings based on current user's subscription
- Added a new Typescript class and a new Angular Service based on this class that consumes the new endpoint and implement a couple of helper methods to check if features are enabled and the limitation to restrict the behaviour of the UI
- Adjusted the app's code to use the new features services to check if features are enabled or some limits have been reached 

### Steps to test
- Checkout the branch and run the project
- Switch between the different subscription plans to see if the limitations are all in place
- Test the app in general to see if the there are any issues that might have been introduced buy these changes

Resolves: [LBSD-2785]

Thanks for checking!

[LBSD-2785]: https://sofab.atlassian.net/browse/LBSD-2785?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
